packages/features/ee/teams/pages/team-members-view.tsx

```diff
@@ -27,13 +27,25 @@ const MembersView = () => {
     },
   });
 
+  const {
+    data: teamMemebers,
+    isLoading: memberLoading,
+    hasNextPage,
+    fetchNextPage,
+    isFetchingNextPage,
+  } = trpc.useInfiniteQuery(["viewer.teams.getMembers", { teamId: Number(router.query.id), limit: 5 }], {
+    getNextPageParam: (lastPage) => lastPage.nextCursor,
+  });
+
```

Comment:
Created a new tRPC endpoint for members. This bug came up with a team that had 20 members. By breaking by paginating this query we should avoid hitting Vercel's 4MB limit.